src/components/organizations/owners.ts

```diff
@@ -10,9 +10,11 @@ export const owners = [
   'Darlington Borough Council',
   'Department for Business, Energy & Industrial Strategy',
   'Department for Education',
+  'Department of Energy Security & Net Zero',
   'Department for Environment, Food & Rural Affairs',
-  'Department for International Trade',
+  'Department for Business and Trade',
```

Comment:
Can we make this 'Department for Business & Trade' rather than 'Department for Business and Trade' to bring it in line with the name on https://www.gov.uk/government/organisations/department-for-business-and-trade

Reply:
the data gets extracted from the platform api and ends up in various spreadsheets elsewhere

Reply:
updated